search/bookmarks/data.cpp

```diff
@@ -0,0 +1,21 @@
+#include "search/bookmarks/data.hpp"
+
+#include <sstream>
+
+using namespace std;
+
+namespace search
+{
+namespace bookmarks
+{
+string DebugPrint(Data const & data)
+{
+  ostringstream os;
+  os << "Data [";
```

Comment:
Bookmark data?
The information about the namespace is lost when printing.

Reply:
We usually do not print namespace info in DebugPrints. IMHO, it's enough to provide class name, because log messages usually contain file name and line number, so it's more than enough to identify what happened.